extensions/safari/src/utils.ts

```diff
@@ -58,9 +58,14 @@ export const getUrlDomain = (url: string) => {
 };
 
 export const getFaviconUrl = (domain: string | undefined) => {
-  if (domain) {
-    return `https://www.google.com/s2/favicons?sz=64&domain=${encodeURI(domain)}`;
+  if (!domain) {
+    return Icon.Globe;
   }
+
+  return {
+    source: `https://www.google.com/s2/favicons?sz=64&domain=${encodeURI(domain)}`,
```

Comment:
This is a nice trick to get the favicon! 🤘

Reply:
It's used in a few extensions. Arguably we could support this via the API.